assets/freedesktop/org.zealdocs.Zeal.appdata.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.zealdocs.Zeal.desktop</id>
+  <name>Zeal</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0</project_license>
+  <summary>Documentation browser</summary>
+  <description>
+    <p>Zeal is a simple offline documentation browser inspired by Dash.</p>
+  </description>
+  <url type="homepage">https://zealdocs.org/</url>
+  <url type="bugtracker">https://github.com/zealdocs/zeal/issues</url>
+  <url type="help">https://zealdocs.org/usage.html</url>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://i.imgur.com/qBkZduS.png</image>
+    </screenshot>
+  </screenshots>
+  <provides>
+    <id>zeal.desktop</id>
+  </provides>
+  <releases>
+    <release date="2018-02-18" version="0.6.0">
```

Comment:
Including changelog will require writing it before tagging a release. Something I tried to avoid in the past.

Do package managers show this to user when a new version is available?

Reply:
I think changelog itself isn't important, but current version is being shown in application store/flatpak CLI. Not a hard requirement yet for upstream-shipped appdata. I suppose I could make it a proper template?

Reply:
Hum, not sure how to proceed with this. Would having additional variable in top-level Cmakelists to hold release date be acceptable to you?

Reply:
@trollixx Some package manager do show the changelog, Discover for example:
https://pointieststick.wordpress.com/2018/02/23/this-week-in-discover-part-7/

But it's not a hard requierement, just a nice-to-have :)

Reply:
In the perfect world release notes should be generated from somewhere, but that feels like an overengineering in Zeal's case. Whatever is the best option according to you guys, let's do that. I didn't know about this appdata thing up until a few days back.

The only thing with 0.6.0 is that it's already tagged, so I guess the next real lease will be 0.6.1.